packages/@sanity/components/src/buttons/styles/DefaultButton.css

```diff
@@ -32,6 +32,7 @@
   @nest &.notRecentlyHovered:focus:not(:hover) {
     overflow: hidden;
 
```

Comment:
There shold be a closing bracket here. Build fails

Reply:
Whaa, don't know what happened there. Will fix!